src/main/docs/guide/ioc/annotationMetadata.adoc

```diff
@@ -1,3 +1,5 @@
+// Doesn't seem appropriate to multi-lang these samples TODO check
```

Comment:
Given this is pseudo code its OK to leave as is for now. Please remove the comment